src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -24,6 +24,17 @@
                 {{ collector.tokenClass|abbr_class }}
             </div>
             {% endif %}
+            {% if 'ROLE_PREVIOUS_ADMIN' in collector.roles %}
+                <div class="sf-toolbar-info-piece">
+                    <b>Switch to user</b>
+                    <a href="?_switch_user=_exit" title="Switch back">Exit</a>
+                </div>
+            {% elseif 'ROLE_ALLOWED_TO_SWITCH' in collector.roles %}
```

Comment:
Exact. Added to the todo list. That goes with the customization of the request parameter.